install.sh

```diff
@@ -164,7 +165,7 @@ stty $stty_orig     # restore terminal setting.
 #####################################################################################################
 echo "$passwd" | sudo -S $SYSTEM_PACKAGE_MANAGER_UPDATE
 echo "$passwd" | sudo -S $SYSTEM_PACKAGE_MANAGER_INSTALL $SYSTEM_PACKAGE_SET
-echo "$passwd" | sudo -S pip install $PIP_PACKAGE_SET
+echo "$passwd" | sudo -S pip install $PIP_INSTALL_CMD $PIP_PACKAGE_SET
```

Comment:
This doesn't make sense. `pip install` should be part of `PIP_INSTALL_CMD`. In Gentoo case `PIP_INSTALL_CMD` shall be overridden and set to `pip install --user` .